addons/web/static/tests/web_test_helpers.js

```diff
@@ -29,6 +29,7 @@ import { defineModels } from "./_framework/mock_server/mock_server";
  */
 
 export { findComponent, mountWithCleanup } from "./_framework/component_test_helpers";
+export { getPickerCell, getPickerApplyButton } from "./_framework/datetime_test_helpers";
```

Comment:
Why?

Reply:
All generic test helpers should be imported from this file and not from the _framework files

Reply:
but i moved it to the datetime tests folder